india_compliance/hooks.py

```diff
@@ -238,6 +238,12 @@
     "Bill of Entry",
 ]
 
+scheduler_events = {
+    "all": [
```

Comment:
note: this is is supposed to trigger every 4 minutes, but it may trigger as often as every 1 minute, based on user configuration. if more control is required, we can switch to cron expression